lib/rubocop/cop/chef/style/use_platform_helpers.rb

```diff
@@ -0,0 +1,59 @@
+#
+# Copyright:: 2019, Chef Software, Inc.
+# Author:: Tim Smith (<tsmith@chef.io>)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+module RuboCop
+  module Cop
+    module Chef
+      module ChefStyle
+        # Use the platform?() and platform_family?() helpers instead of node['platform] == 'foo'
+        # and node['platform_family'] == 'bar'. These helpers are easier to read and can accept multiple
+        # platform arguments, which greatly simplifies complex platform logic.
+        #
+        # @example
+        #
+        #   # bad
```

Comment:
uhmmmm. Is this a `WARN` or and `ERROR`? Meaning if this is a recommendation to make the code better or a thing the user /must/ change because it is a /bad/ usage?

It is a bit confusing to call a few cops bad/good. Maybe we can chat about it over zoom/standup. (thought just keep merging things since that has been the pattern we follow)

Reply:
At the moment everything is a warning. We're not failing builds with any new cops until April 2020 when some of these will move to failures and some will remain at the refactor level.